<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kategorieeingabe</class>
 <widget class="QDialog" name="Kategorieeingabe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kategorien </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kategorie:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="einlesenKategorie">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonHinzufuegen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
    </font>
   </property>
   <property name="text">
    <string>Hinzufügen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonWeiter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Bold</family>
    </font>
   </property>
   <property name="text">
    <string>Fertig und weiter zur Spielerübersicht</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonZurueck">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
    </font>
   </property>
   <property name="text">
    <string>Zurück</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableKategorie">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
